internal/config/main.go

```diff
@@ -140,7 +140,7 @@ func (c *Config) AutoDetect() error {
 		return err
 	}
 	c.autoscalingVersion = hpaVersion
-	c.logger.Info("In Autodetect, Set HPA version to [", c.autoscalingVersion, "] from [", hpaVersion, "]")
+	c.logger.V(2).Info("In Autodetect, Set HPA version to [", c.autoscalingVersion.String(), "] from [", hpaVersion.String(), "]")
```

Comment:
Aren't `autoscalingVersion` and `hpaVersion` the same value?

Also - the first string is a message but the remaining in the list are key value pairs, leaving the values as `] from [` and `]` - is this what we want?

I tried to fix this when I was working on an operator change and came up with:
```
c.logger.Info("In Autodetect, Set HPA version", "from", c.autoscalingVersion, "to", hpaVersion)
```

Reply:
+1, after the log message follows a list of key-value pairs. so it should be something like `info("message line", "param1-name", param1-value)`

Regarding the `v(2)`. It makes sense to use the lowest value, however if the codebase already uses v(2) as a debug level then 2 is fine.